docs/guides/transactions/running-background-jobs.md

```diff
@@ -1,85 +1,10 @@
 # Running Background Jobs
-
+ 
 ## Introduction
 
-This article describes useful patterns to run background jobs in Starcounter applications which require access to the database. In regards to the current implementation in Starcounter 2.x, it is important to keep in mind how transactions and deleting of objects work and thus avoid jobs that run for a long time, or possible forever.
-
-When you perform reads from the database, an implicit read-only transaction is created for you carefully by Starcounter. I.e., `Main` of Starcounter application, delegates passed to `Scheduling.RunTask` and URI handling delegates are all wrapped into such an implicit read-only transaction. The transaction allows you to do reads from the database. If a job inside that transaction is running forever, deleted objects will only be marked as deleted but not removed from the database until the code-host is restarted or until the delegate is done and some other transaction performs a commit afterwards. This can have implications of space \(on disc\) needed for the image-files since they will continue to grow even if objects are deleted.
-
-What it means practically is that spawning background tasks by running an infinite loop with a timer inside `Scheduling.RunTask`, URI-handling delegates and `Main` isn't a good idea. This is not only preventing object from purging and will steal computing resources, but is also considered a bad pattern of a background job in general. The correct pattern uses inversion of control, when a timer object invokes the action, and is explained below. The main point is to keep transaction scopes short.
-
-## Long-running Threads
+Code that is provided via Starcounter API (transactions, SQL, request/response handling, Web sockets handling, etc.) should execute in a special context. Such context is already given inside the `Main()` function, view-model handles and handlers (for example, `Handle.GET`). In other situations, like in separately created background .NET threads or tasks, to be able to execute Starcounter code, one should schedule it using `Scheduling.RunTask()`.
```

Comment:
A couple of minor points on this paragraph:
* `Main` is a method not a function
* We usually write the names of methods without their parameters, so `Scheduling.RunTask` not `Scheduling.RunTask()`. This is just a convention, so do what you want with it
* "A special context" is a little vague. I know that you've explained in more clearly in person before. Could you add a sentence on what this context does? 

Other than that, I think it's excellent!